.github/workflows/security-audit.yml

```
@@ -0,0 +1,266 @@
+  pull_request:
+    branches: [main, develop]
+  schedule:
+    # 每天 UTC 02:00 运行安全扫描
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+
+jobs:
+  security-audit:
+    name: Security Vulnerability Audit
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)